types/webvr-api/index.d.ts

```diff
@@ -163,9 +186,9 @@ interface VRFrameData {
 }
 
 declare var VRFrameData: {
-    prototype: VRFrameData
-    new(): VRFrameData
-}
+    prototype: VRFrameData;
+    new(): VRFrameData;
+};
 
 interface VRPose {
```

Comment:
I think i've done it precisely the same as they are in https://github.com/Microsoft/TypeScript/blob/master/lib/lib.es6.d.ts

So although I agree it's a weird pattern I see no reason not to match the official defs to avoid confusion.

Does that seem sound?

Reply:
Fair enough. :)